content/applications/websites/ecommerce/customer_accounts.rst

```diff
@@ -2,42 +2,128 @@
+
+After logging in, customers can access their documents by clicking their username in the top-right
+corner of the screen selecting :guilabel:`My Account`. From there, they can view all their
+documents in one place, such as :guilabel:`quotations`, :guilabel:`orders`,
+:guilabel:`invoices` and more.
 
 .. image:: customer_accounts/account-log.png
-   :align: center
-   :alt: Customer account log-in
+   :alt: Customer account access
+
+.. tip::
+   Similarly to the rest of the website, the customer account page can be customized with building
+   blocks and other features through the :doc:`website builder <../website/web_design>`.
+
+.. _ecommerce/customer_accounts/account-creation:
+
+Customer account creation
+=========================
+
+You can choose whether customer accounts and document access are available to everyone or restricted
+to invited users only. To do so, go to :menuselection:`Website --> Configuration --> Settings`,
+then scroll down to the :guilabel:`Privacy` section. Under :guilabel:`Customer Account`, select one
+of the following options:
+
+- :guilabel:`On invitation`: Customers can only create an account if the website owner sends them
+  an invitation.
+- :guilabel:`Free sign up`: Every website visitor can create an account and sign in. They will
+  get access to the :doc:`portal <../../general/users/portal>` by default.
+
+If account creation is restricted to invited customers only, go to :menuselection:`Website -->
+eCommerce --> Customers`, switch to the :guilabel:`List` view, and select the customers who
+should gain access to the :doc:`portal <../../general/users/portal>`.
+Click the :icon:`fa-cog` :guilabel:`Actions` button, then :guilabel:`Grant portal access`.
+In the :guilabel:`Portal Access Management`, review the selected customers and click
+:guilabel:`Grant Access` to confirm.
+
+Once done, the relevant customers receive an email confirming their account creation, including
+instructions on setting a password and activating their account.
 
 .. note::
-   Customers can only have an account if the
-   :ref:`sign in/up at checkout <ecommerce/checkout/policy>` option allows for account creation.
+   - When selecting the :guilabel:`Free sign up`, a clickable :guilabel:`Don't have an account?`
+     link appears under the login form on the website.
+   - The :guilabel:`On invitation` option is especially useful for B2B businesses that prefer
+     to keep prices hidden on the website. Only customers who have been granted portal access are
+     able to view their pricing.
 
 .. tip::
-   Similarly to the rest of the website, the customer account page can be customized with **content
-   blocks** and other features through the **website builder**.
-
-.. seealso::
-   :doc:`/applications/general/users/portal`
+   It is possible to configure a website form with a :guilabel:`Create a Customer` :ref:`action
+   <website/building_blocks/form>` to automatically create a customer record in the backend when
+   filled in.
```

Comment:
```suggestion
   It is possible to configure a :ref:`website form <website/building_blocks/form>` with a :guilabel:`Create a Customer` action
   to automatically create a customer record in the backend when
   filled in.
```
I would move the ref under "website form" since it leads to the Form heading. Or you could leave the ref on "action" and make it lead to the "Action" heading instead :)